extension-ui/oops.scss

```diff
@@ -0,0 +1,122 @@
+body {
+  background-color: #f0f4f8;
```

Comment:
the following comments relevant for all this file
1. colors and font-family should come from variables file
2. don't use px's
3. use scss structure
4. you can't write styles on elements like body, h1, p etc, you should add classes or id's and style them